packages/esm-patient-search-app/src/compact-patient-search/compact-patient-search.scss

```diff
@@ -0,0 +1,16 @@
+@import '../root.scss';
+
+.patientSearchBar {
+  width: 50vw;
+  position: relative;
+}
+
+.floatingSearchResultsContainer {
+  position: absolute;
+  overflow-y: auto;
+  box-shadow: 0 $spacing-03 $spacing-05 $ui-03;
+  z-index: 99;
+  border: 0 0.063rem 0.063rem 0.063rem solid $ui-03;
```

Comment:
is rem better than pixels for border width? this renders to 1px, but I've read that if it calculates to less than a pixel some browsers won't render it?